src/Render.cc

```diff
@@ -57,13 +64,26 @@ namespace drafter {
             case JSONRenderFormat:
             {
                 refract::RenderJSONVisitor renderer;
-                refract::IElement* element = MSONToRefract(attributes);
+
+                refract::IElement* element = MSONToRefract(*attributes);
+
+                if (!element) {
+                    return body;
+                }
+
```

Comment:
> delete input element

It instead of modifying the input element and returning it (which it was doing previously), it now deletes the input element and returning a new element. That way, it is a bit clear.

If you really think this is wrong, please make a separate PR for this later. Let's not mix things up.